src/test/java/InvalidLoginTests.java

```diff
@@ -0,0 +1,25 @@
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import Pages.LoginPage;
+
+public class InvalidLoginTests extends BaseTest{
+    LoginPage loginPage;
+    @BeforeMethod
+    void setup(){
+        loginPage= new LoginPage(getThreadDriver());
+        getThreadDriver().get(url);
+    }
+    @Test
+    public void invalidPassword() throws InterruptedException {
+        loginPage.provideEmail("incorrectEmail").providePassword("incotrerctPwd").clickSubmit();
+        Thread.sleep(3000);
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.